fix: include type definitions generated from JSDoc (#195)
eik-lib/node-client: README.md

# @eik/node-client

> This is an Eik utility for servers running on Node. With it you can:

Reviewer: ```suggestion
This is an Eik utility for working with [@eikservice](https://github.com/eik-lib/service) and your configuration. With it you can:
```
This feels a bit vague, as there is no other option than to run an Eik server on node using the `service` module?

- generate different URLs to assets on an Eik server depending on environment (development vs production).
- get the import maps you have configured in `eik.json` from the Eik server, should you want to use them in the HTML response.

## Install

```sh
npm install @eik/node-client
```

## Usage

The most common use case for this module is linking to a file.

> When developing you typically want to use a local version of the file, then link to the published version on Eik when running in production.

Reviewer: This key difference between development and production I think could be emphasised a bit more, as having the client in development mode means it does something entirely different than in production 🤔 

I see it mentioned in the method docs, which is really good, I was just thinking for my self I would read this passage and just glance over the method docs. Maybe we could write something about this difference in behaviour here too.
